docs/troubleshooting/detections/detection-rules.asciidoc

```diff
@@ -83,7 +83,7 @@ CAUTION: Turning off `autocomplete:useTimeRange` could cause performance issues
 [%collapsible]
 ====
 
-A warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]) and message appear for fields with <<fields-with-conflicting-types,type conflicts>> across multiple indices or are <<unmapped-field-conflict,unmapped fields>>. You can learn more about the conflict by hovering over the field. After you select it, the warning message appears beneath the field.
+A warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]) and message appear for fields with <<fields-with-conflicting-types,type conflicts>> across multiple indices or that are <<unmapped-field-conflict,unmapped fields>>. You can learn more about the conflict by clicking the warning message. 
```

Comment:
Tiny suggestion, otherwise LGTM 🚀 

```suggestion
A warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]) and message appear for fields with <<fields-with-conflicting-types,type conflicts>> across multiple indices and <<unmapped-field-conflict,unmapped fields>>. You can learn more about the conflict by clicking the warning message. 
```

Reply:
I think I want to keep the "or" since it's inclusive of both scenarios but doesn't imply that both conditions need to be met (i.e. a field needs to have type conflicts _and_ be unmapped) for the warning to display. 

How does the following revision look

```suggestion
A warning icon (image:images/field-warning-icon.png[Field conflict warning icon,13,13]) and message appear for fields with <<fields-with-conflicting-types,type conflicts>> across multiple indices or  fields that are <<unmapped-field-conflict,unmapped>>. You can learn more about the conflict by clicking the warning message. 
```

Reply:
Makes sense – this looks good!